README.md

```diff
@@ -250,7 +251,7 @@ The remaining build tools are available from the Ports Collection, and can be in
 To build Julia, simply run `gmake`.
 (Note that `gmake` must be used rather than `make`, since `make` on FreeBSD corresponds to the incompatible BSD Make rather than GNU Make.)
 
-It's important to note that the `USE_SYSTEM_*` flags should be used with caution on FreeBSD.
+It is important to note that the `USE_SYSTEM_*` flags should be used with caution on FreeBSD.
```

Comment:
(same nit as above)

Reply:
It's especially important on FreeBSD, because while you may run into some issues using `USE_SYSTEM_*` on other platforms, it's likely that your Julia build just straight up won't work if you do it on FreeBSD.

Reply:
"As mentioned above" seems okay to me.